rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/v1/CpcFileControllerV1.java

```diff
@@ -89,22 +93,30 @@
 	}
 
 	/**
-	 * Updates a file's status to processed in the database
+	 * Updates a file's status in the database
 	 *
 	 * @param fileId Identifier of the file needing to be updated
 	 * @return Message if the file was updated or not
 	 */
 	@RequestMapping(method = RequestMethod.PUT, value = "/file/{fileId}",
 			headers = {"Accept=" + Constants.V1_API_ACCEPT} )
-	public ResponseEntity<String> markFileProcessed(@PathVariable("fileId") String fileId) {
+	public ResponseEntity<String> updateFile(@PathVariable("fileId") String fileId,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document the parameter(s): request [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)